From 256cc3d5a0a3fe07a0a3cd9376f89cb24bcc0eef Mon Sep 17 00:00:00 2001
From: Michal Schmidt <mschmidt@redhat.com>
Date: Tue, 10 Jan 2012 22:03:49 +0100
Subject: [PATCH 202/217] logs-show: fix missing newline in short output

Ellipsized messages were printed without a newline.
(cherry picked from commit 52beb2c379674e066d6cf50d59874478403e4da4)
---
 src/logs-show.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/logs-show.c b/src/logs-show.c
index 2fe205f..8d728a6 100644
--- a/src/logs-show.c
+++ b/src/logs-show.c
@@ -237,7 +237,7 @@ static int output_short(sd_journal *j, unsigned line, bool show_all, bool monoto
                 if (!e)
                         printf(": %.*s\n", (int) message_len, message);
                 else
-                        printf(": %s", e);
+                        printf(": %s\n", e);
 
                 free(e);
         } else
-- 
1.7.8.3

